src/z3c/flashmessage/README.txt

```diff
@@ -12,19 +12,24 @@ To send a message to the current user, you can use the session-based message
 source. Let's set one up:
 
 >>> from z3c.flashmessage.sources import SessionMessageSource
+>>> from __future__ import unicode_literals
 >>> source = SessionMessageSource()
 
->>> source.send(u'The world will come to an end in 40 seconds!')
+>>> source.send('The world will come to an end in 40 seconds!')
 
 The source allows to list all current messages:
 
 >>> m = list(source.list())
 >>> m
 [<z3c.flashmessage.message.Message object at 0x...>]
 >>> m[0].message
-u'The world will come to an end in 40 seconds!'
->>> m[0].type
-u'message'
+'The world will come to an end in 40 seconds!'
```

Comment:
Why it is not required to have the `if sys.version_info < (3, 0)` here like for `m[0].type`? (see below)